example/methods_directory/common_parameters/precision.json

```diff
@@ -0,0 +1,4 @@
+{
+    "energyConvergence": "1e-7",
+    "gridSize": "coarse"
+}
```

Comment:
I was thinking that having a common parameter dir for functionality that is similar across methods could be useful. This could be where things like energy convergence can be located, as it is not specific to one method, but could work with multiple methods.

Reply:
I like the intent, however, the definition is unclear as it stands now. I think each of these parameters easily deserve their own data structures. For the energy, we would need to reuse the energy property schema available elsewhere in ESSE to also have the units. For the grid size, we'd need more parameters too - type of grid, size of grid elements etc. And if we are changing the grid here, we'd need to adjust the implementation in pseudopotential accordingly method as well. Perhaps we should create a separate task to deal with these?

Reply:
I think a separate task for various parameters within dft methods would be best, as they likely require more coding to completely integrate into the code.